Luminescent/src/astechzgo/luminescent/entity/Player.java

```diff
@@ -191,15 +191,16 @@ else if(KeyboardUtils.isKeyDown(Constants.KEYS_MOVEMENT_UP))
 		double x = 0;
 		double y = 0;
 		
-		boolean bx = false;
-		boolean by = false;
-		
+	
```

Comment:
I did some testing with this, and it looks like you're running into the same problem I am.  If you use this, it will not work.
```
[
	{
		"x": "CAMERA_WIDTH / 45",
		"y": "CAMERA_HEIGHT / 25",
		"width": "CAMERA_WIDTH - (CAMERA_WIDTH / 45 * 2)",
		"height": "CAMERA_HEIGHT - (CAMERA_HEIGHT / 25 * 2)"
	},
	{
		"x": "CAMERA_WIDTH / 45",
		"y": "CAMERA_HEIGHT / 25 + (CAMERA_HEIGHT - (CAMERA_HEIGHT / 25 * 2)) - 1",
		"width": "CAMERA_WIDTH - (CAMERA_WIDTH / 45 * 2)",
		"height": "CAMERA_HEIGHT - (CAMERA_HEIGHT / 25 * 2)"
	}
]
```